posthog/templates/signup_to_organization_company.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'layout.html' %}
```

Comment:
Think this could be done in the React frontend already, but it could also be an improvement

Reply:
Moving all of signup and login is definitely the goal eventually, but for consistency now it made sense to go with the Django template here.